src/index.html

```diff
@@ -9,6 +9,81 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header container">
+      <img
+        src="images/logo.png"
+        alt="moyo"
+        class="header__logotype"
+        >
+      <nav class="nav">
+        <ul class="nav__list">
+          <li>
```

Comment:
```suggestion
          <li class="nav__item">
```
https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#correct-class-naming